FlightJournal.Web/Migrations/FlightContext/Configuration.cs

```diff
@@ -216,6 +216,12 @@ protected override void Seed(FlightJournal.Web.Models.FlightContext context)
             }
 
 
+            if (!context.ListenerAreas.Any())
+            {
+                context.ListenerAreas.Add(new ListenerArea {Latitude = 55.84, Longitude = 10.63, Radius = 220});
```

Comment:
I propose a Note field also. 
The Seed data will in all practicality become one to one with what we setup on live i presume atm.